### DISCUSSION TITLE
Installing netbox-secrets in docker netbox and asking for secrets.js ?

### DISCUSSION THREAD

**Discussion by PVM-NL** [Onemind-Services-LLC/netbox-secrets#37] [General]
Hi,

Today i was also testing your plugin on my docker environment and i notice that maybe something is not copied.
Im not sure if this is a bug so i start a discussion first.

Mainly what i do is add the plugin to the plugin_requirements.txt and in /configuration/  i add the pluginname to the plugin.py
Then i build the docker and bring it up. After that i go to the website on my desktop and browse in netbox. But as soon as go to the secret page it say missing "static/netbox_secrets/secrets.js". 

So to fix the app on docker i went in the bash mode and created the file myself.

**Comment by abhi1693** [comment]
You also need to run `./manage.py collectstatic` in your virtual environment. I believe this is not done automatically in the docker repo for netbox

**Reply by PVM-NL** [reply]
That was indeed the case, thanks
